library/ix-dev/community/audiobookshelf/templates/_audiobookshelf.tpl

```diff
@@ -0,0 +1,40 @@
+{{- define "audiobookshelf.workload" -}}
+workload:
+  audiobookshelf:
+    enabled: true
+    primary: true
+    type: Deployment
+    podSpec:
+      hostNetwork: {{ .Values.audiobookshelfNetwork.hostNetwork }}
+      containers:
+        audiobookshelf:
+          enabled: true
+          primary: true
+          imageSelector: image
+          securityContext:
+            runAsUser: {{ .Values.audiobookshelfRunAs.user }}
+            runAsGroup: {{ .Values.audiobookshelfRunAs.group }}
+          {{ with .Values.audiobookshelfConfig.additionalEnvs }}
```

Comment:
```suggestion
          env:
            PORT: {{ .Values.audiobookshelfNetwork.webPort }}
            CONFIG_PATH: /config
            METADATA_PATH: /metadata
          {{ with .Values.audiobookshelfConfig.additionalEnvs }}
```
With this we can sync the external with the internal port to avoid any confusions.
Also if we don't do that and we allow hostNetwork, it will try to bind the port 80 on the host where the WebUI of truenas runs on.

I've also locked the paths to the specific paths, to avoid any misconfiguration that could lead to weird situations!

Reply:
Ah!  I was confused how that worked for the others apps, checking the host network didn't seem to operate any differently, but now that makes sense!  Thank you!